server/files/javascript/emoji.js

```diff
@@ -20,7 +20,7 @@ semantic.emoji.ready = function() {
   var addEmojiChunk = function(){
     var $button = $(this),
         category = $button.attr('data-category'),
-        chunkpos = $button.attr('data-chunkpos'),
+        chunkpos = parseInt($button.attr('data-chunkpos')),
```

Comment:
It is recommended to always specify radix at 2nd argument of parseInt.
Because if 1st argument is something like `023`, radix is infered as 8, not 10.

Reply:
oups..thanks. Fixed by 2353a27a9bdc21aca190ffc1610fec4ab3cbc101